templates/content/oe-theme-helper-in-page-navigation--node--oe-person.html.twig

```diff
@@ -6,4 +6,4 @@
  * @see ./modules/oe_theme_helper/templates/oe-theme-helper-in-page-navigation.html.twig
  */
 #}
-{% include "oe-theme-helper-in-page-navigation--with-image.html.twig" %}
+{% extends "oe-theme-helper-in-page-navigation--with-image.html.twig" %}
```

Comment:
Please use proper referral, with "@oe_theme/content/oe-theme-helper-in-page-navigation--with-image.html.twig"